ackermann_steering_controller/CMakeLists.txt

```diff
@@ -1,11 +1,6 @@
 cmake_minimum_required(VERSION 3.0.2)
 project(ackermann_steering_controller)
 
-# Default to C++14
-if(NOT CMAKE_CXX_STANDARD)
```

Comment:
@v4hn @jspricke Very excited about the ROS-O effort. Since I've seen a few of these PRs around, for my understanding, if the `set(CMAKE_CXX_STANDARD` is guarded with `if(NOT CMAKE_CXX_STANDARD)`, should it still be removed?

Reply:
Thank you, all clear. I was wondering about older operating system releases as well (as in other projects we don't branch for every release / try to keep one branch for as many distros as possible).

Reply:
C++14 "is the default in GCC 6.1 up until GCC 10 (including)"
https://gcc.gnu.org/projects/cxx-status.html#cxx14

Ubuntu 18 has GCC 7.4:

https://packages.ubuntu.com/search?suite=all&searchon=names&keywords=gcc

ROS Melodic targets C++14:

https://www.ros.org/reps/rep-0003.html#melodic-morenia-may-2018-may-2023